### PR TITLE
Merge release branch into master

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: scripts/mkreleaselog [hunk 1]
```diff
@@ -1,31 +1,11 @@
...
-    whyrusleeping
-    Kubuxu
-    jbenet
-    Stebalien
-    marten-seemann
-    hsanjuan
-    lucas-clemente
-    warpfork
-)
+AUTHORS=(filecoin-project)
```
Comment (ribasushi) [inline scripts/mkreleaselog:8]: @Stebalien @magik6k I think the above is a mistaken revert of https://github.com/filecoin-project/lotus/pull/5826
I.e. in `master` we want the full list, no?
cc @BigLep @arajasek

**Review by ribasushi — APPROVED** [review]
LGTM, left a nit comment for when folks wake up

File: scripts/mkreleaselog (hunk 1 above)
Comment (magik6k) [reply scripts/mkreleaselog:8]: Nope, this is a new commit -> https://github.com/filecoin-project/lotus/pull/6583/commits/1411cff69e3dd05dc2be073e104606ab88d34490

Comment (ribasushi) [reply scripts/mkreleaselog:8]: Right, I believe *that* commit is a mistake

**Pull request merged**